index.html

```diff
@@ -0,0 +1,133 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Microverse</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  <script
+      src="https://kit.fontawesome.com/e520ead38a.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="top">
+        <i class="fa-brands fa-facebook-f social"></i>
+        <i class="fa-brands fa-twitter social"></i>
+        <ul class="top-list">
+          <li class="top-items">English</li>
+          <li class="top-items">MyPage</li>
+          <li class="top-items">Logout</li>
+        </ul>
+      </div>
+      <div class="main-nav">
+        <img src="./images/microverseLogo.png" class="logo" alt="Microverse Logo">
+        <ul class="menu-mobile" id="menu">
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="./about.html">About</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">Program</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">Join</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">Sponsor</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">News</a></li>
+          <i class="fa-solid fa-xmark close" onclick="closeMenu()"></i>
+        </ul>
+        <ul class="menu">
+          <li class="menu-list"><a href="./about.html">About</a></li>
+          <li class="menu-list"><a href="#">Program</a></li>
+          <li class="menu-list"><a href="#">Join</a></li>
+          <li class="menu-list"><a href="#">Sponsor</a></li>
+          <li class="menu-list"><a href="#">News</a></li>
+          <button type="button" class="button">CC Campaign</button>
+        </ul>
+      </div>
+      <i class="fa-solid fa-bars open" onclick="openMenu()"></i>
+    </nav>
+  </header>
+  <main>
+    <section class="body">
+      <article class="main-text">
+        <p class="intro">"Hello! Sharing world"</p>
+        <h1 class="title">Become an international software developer, from anywhere.</h1>
+        <article class="information">
+          <p>Gain the skills you need to get hired. Learn online, connect to life-changing opportunities. Pay $0 until you land a job!</p>
+        </article>
+      </article>
+    </section>
+
+<!-- -----------------------------main program-------------------------------- -->
+
+    <section class="program">
+      <div class="program-container">
+        <h2 class="main-program">Main Program</h2>
+        <div class="line"></div>
+        <div class="method-container">
```

Comment:
- [ ] Please check the alignment on mobile versions. It seems a bit off when compared to the design. ✍️

| Your design                                                             | Template                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/211502523-5832b1b9-cc51-48c5-ad3b-123efbb23694.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/211502535-1dd621cd-5e48-4683-afb0-79899f1817c2.png" width="300">|

Reply:
Fixed.